lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -45,6 +46,7 @@ def meraki_argument_spec():
                 timeout=dict(type='int', default=30),
                 org_name=dict(type='str', aliases=['organization']),
                 org_id=dict(type='str'),
+                output_version=dict(type='str', choices=['old', 'new'], default='new'),
```

Comment:
It's probably better to use more specific names than `old` and `new`, in case you want to add something else in the future. Using `new2` or `newer` will look really strange ;)

Reply:
Old, new, newer, newest, newestAF...I dont see the big deal :)

Any ideas on replacements? I commented below with the possibility of snake and camel. But those aren’t obvious.

Reply:
That's a very good question. Maybe an integer? Version 1 is the old format, version 2 the new. Not a very helpful notation, though. And specifying a long string is also annoying.

Reply:
I removed this option and moved it to an env var instead.